.github/working_agreement.md

```diff
@@ -0,0 +1,171 @@
+- **Be respectful**: We are a world-wide community of professionals, and we conduct ourselves professionally. Disagreement is no excuse for poor behavior and poor manners. Disrespectful and unacceptable behavior includes, but is not limited to:
+  - Violent threats or language.
+  - Discriminatory or derogatory jokes and language.
+  - Posting sexually explicit or violent material.
+  - Posting, or threatening to post, people's personally identifying information ("doxing").
+  - Insults, especially those using discriminatory terms or slurs.
+  - Behavior that could be perceived as sexual attention.
+  - Advocating for or encouraging any of the above behaviors.
+- **Understand disagreements**: Disagreements, both social and technical, are useful learning opportunities. Seek to understand the other viewpoints and resolve differences constructively.
+- This code is not exhaustive or complete. It serves to capture our common understanding of a productive, collaborative environment. We expect the code to be followed in spirit as much as in the letter.
+
+## How we plan together
+
+### Work Items
+
+- We will track our work in GitHub
+- For the most part, our hierarchy will follow Epic > Story format. However, this is not a hard requirement as some issues do not nicely tie to epics.
+- We will track Risk work items outside of GitHub so that we may easily manage them independently; however, we may choose to create an issue for them should that make sense.
+
+|  | Sizing | Definition |
+|--|--------|------------|
+| **Epic** | Up to the lifetime of the project | Business initiative for a stakeholder to accomplish |
+| **Story** | Completable within a sprint | Consists of multiple tasks |
+| **Bug** | Completable within a sprint | Production blocking bugs are prioritized |
+
+#### User Story Guidelines
+
+> Sourced from <https://www.scrumtraining.com>
+
+##### User Story Guideline
+  As a *[persona]*, I need *[ask]*, so that *[end desire]* OR
+  Who, What, Why, When
+  
+##### Acceptance Criteria/Tests
+
+   User can *[select/operate] [Feature/Function]* so that *[output]* is *[visible/complete/etc.]*  
+   Verify that...
+
+##### I.N.V.E.S.T. in User Stories
+
+**I**ndependent - Can it be developed independently of other stories?  
+**N**egotiable - Is the scope negotiated to enable completion?  
+**V**aluable - Is the value to the user or customer clear?  
+**E**stimatable - Can the work be estimated? Are there unknowns, dependencies, barriers? Do we lack domain or technical knowledge?  
+**S**ize appropriately - Can it be completed in the iteration?  
+**T**estable - What are the tests to know the work is done?
+
+##### User Story Issues to Avoid
+
+- Describing a task
+- UI too soon
+- Write in an inactive voice
+- Splitting too often in the iteration
+- Thinking too far ahead
+- Interdependent Stories
+- Too many details
+- Goldplating
```

Comment:
I had to look this up, not sure how many folks have encountered this term. A link to the wikipedia article would be helpful - https://en.wikipedia.org/wiki/Gold_plating_(project_management)